content/quiz_process.md

```diff
@@ -7,19 +7,63 @@
         "2-Team 20-Question", "2-Team Overtime", and "3-Team Overtime".
 
 ### 3-Team Quiz
+- There will be 20 numbered questions in a quiz.
+- In the event the quiz ends in a tie, additional units of three overtime questions each will be used until the tie is broken.  Two-team rules will apply in overtime if only two teams are involved.  During overtime, all the numbered questions will be of a different question type.
+- A toss-up question is the result of an error by one team.  The two remaining teams will have the opportunity to jump, but the team making the error is ineligible for that question.
+    - If question 1 to 15 is erred, the next numbered question will be a toss-up question. 
+    - If question 16 is erred, the toss-up question will be question 16A.  If 16A is erred, then 16B is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred. 
+- If toss-up question 2 to 15 is erred, then the next numbered question is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred. 
+- If question 16 is a toss-up question and is erred, 16A is awarded as a 20-point assigned bonus question to the team who has not erred.  This bonus question goes to the corresponding chair of the team that has not erred.
+- Error points begin at question 17.
+- The first error committed, beginning at question 17, results in a toss-up question, which is numbered 17A, for the two remaining teams.  If there is an error on 17A, the next question will be 17B and will be a 10-point bonus for the team that has not committed an error.  This process applies for the remainder of the quiz.  Three teams must be enlisted to jump on questions 17, 18, 19, and 20.
 
```

Comment:
I left this out too, since I felt it belongs under Scoring "All questions have a value of 20 points when two or three teams are eligible."

Reply:
Agreed. All scoring should be in the scoring section.